config/jobs.txt

```diff
@@ -31,7 +31,7 @@ Ship Engineer=5,5
 Atmospheric Technician=3,2
 
 Medical Doctor=5,3
-Chemist=2,2
+Chemist=1,1
```

Comment:
Config files won't be updated on the server, they need to be edited manually.

@ninjanomnom @Vivalas @monster860